projects/core/src/cx-api/cx-api.service.ts

```diff
@@ -1,21 +1,15 @@
-import { Injectable, Optional } from '@angular/core';
+import { Optional } from '@angular/core';
+import { AuthService } from '../auth';
```

Comment:
each import should use the corresponding barrels - if the barrel exists (lines 2-11)